tools/picard.js

```diff
@@ -247,6 +247,15 @@ function get_module_menu () {
     return menu;
 }
 
+// Our sdk theme only resets widgets inside a window will css name EosWindow. We
+// will take advantage of that to make an adwaita themed topbar with endless
+// themed widgets inside.
+const ResetThemeBox = new Knowledge.Class({
```

Comment:
No need to use `Knowledge.Class` here and in fact it's easier to use `Gtk.WidgetClass`'s `CssName` property:
```js
const ResetThemeBox = new Lang.Class({
    Name: 'ResetThemeBox',
    Extends: Gtk.Box,
    CssName: 'EosWindow',
});
```
Also make sure the names match up.